website/pages/docs/theming/scale-out-theme.mdx

```diff
@@ -0,0 +1,519 @@
+  airbnb: 'rgba(0, 0, 0, 0.15) 0px 2px 8px',
+  mac: 'rgba(0, 0, 0, 0.2) 0px 20px 30px',
+  // More...
+};
+
+export type Shadows = typeof shadows
+
+export default shadows
+```
+
+### 🚀 Spacing
+Setting the size of the whitespace for `padding`, `margin` and `gap` etc.
+
+```ts live=false
+// spacing.ts
+
+export const spacing = {
+  full: '100%',
+  half: '50%',
+  x1: '1em',
+  x2: '2em',
+  x4: '4em',
+  x8: '8em',
+  x16: '16em',
+  // More...
+}
+
+export type Spacing = typeof spacing
+```
+
+### 📏 Sizes
+You can customize the global sizing of components you build for your project.
+These value can be used by the `width`, `height`, and `maxWidth`, `minWidth`, `maxHeight`, `minHeight` styles.
+
+```ts live=false
+// sizes.ts
+import { spacing } from "./spacing"
+
+const largeSizes = {
+  full: "100%",
+  half: "50%",
+  "3xs": "14rem",
+  "2xs": "16rem",
+  xs: "20rem",
+  sm: "24rem",
+  md: "28rem",
+  lg: "32rem",
+  xl: "36rem",
+  "2xl": "42rem",
+  "3xl": "48rem",
+  // More...
+}
+
+const container = {
+  sm: "640px",
+  md: "768px",
+  lg: "1024px",
+  xl: "1280px",
+}
+
+const sizes = {
+  ...spacing,
+  ...largeSizes,
+  container,
+}
+
+export type Sizes = typeof spacing &
+  typeof largeSizes & { container: typeof container }
+
+export default sizes
+```
+
+### 📝 Typography
+To customize Typography options, the theme object supports the following keys:
+
+- `fonts` (font families)
+- `fontSizes`
+- `letterSpacings`
+- `lineHeights`
+- `fontWeights`
+
+```ts live=false
+const typography = {
+  fonts: {
+    body: "system-ui, sans-serif",
+    heading: "Georgia, serif",
+    mono: "Menlo, monospace",
+  },
+
+  fontSizes: {
+    xs: "0.75rem",
+    sm: "0.875rem",
+    md: "1rem",
+    lg: "1.125rem",
+    xl: "1.25rem",
+    "2xl": "1.5rem",
+    // More...
+  },
+
+  letterSpacings: {
+    tighter: "-0.06em",
+    tight: "-0.03em",
+    normal: "0",
+    wide: "0.03em",
+    wider: "0.06em",
+    // More...
+  },
+
+  lineHeights: {
+    shorter: 1.3,
+    short: 1.5,
+    base: 1.7,
+    tall: 1.9,
+    taller: "2",
+    // More...
+  },
+
+  fontWeights: {
+    // Custom in special cases. Like this:
+    // normal: 400,
+    // bold: 700,
+  },
+
+}
+
+export type Typography = typeof typography
+
+export default typography
+```
+
+### 💨 Transition
+You can customize transition like this:
+
+```ts live=false
+// transition.ts
+
+const transitionProperty = {
+  common:
+    "background-color, border-color, color, fill, stroke, opacity, box-shadow, transform",
+  colors: "background-color, border-color, color, fill, stroke",
+  dimensions: "width, height",
+  position: "left, right, top, bottom",
+  background: "background-color, background-image, background-position",
+}
+
+const transitionTimingFunction = {
+  "linear": "cubic-bezier(0, 0, 1, 1)",
+  "ease-in": "cubic-bezier(0.42, 0, 1, 1)",
+  "ease-out": "cubic-bezier(0, 0, 0.58, 1)",
+  "ease-in-out": "cubic-bezier(0.42, 0, 0.58, 1)",
```

Comment:
I think we can reduce the number of examples here. The reader already understood the pattern how to override theme tokens.